source/NanostackRfPhyAT86RF215.cpp

```diff
@@ -1170,6 +1187,17 @@ static void rf_conf_set_cca_threshold(uint8_t percent)
     cca_threshold = MIN_CCA_THRESHOLD + (step * percent) / 100;
 }
 
+static bool rf_conf_set_tx_power(uint8_t percent)
+{
+    uint8_t step = (TXPWR_31 - TXPWR_0);
```

Comment:
Should you add validation that if value is larger than 100 it is capped at 100?

Reply:
Sniffer application is actually doing that check. Should we just ignore invalid value on RF driver and return -1 in callback?